kudos-board-ui/src/pages/homePage.jsx

```diff
@@ -11,7 +11,9 @@ function HomePage() {
   const [isModalOpen, setIsModalOpen] = useState(false);
   const VITE_URL = import.meta.env.VITE_URL;
 
+  // Error handling
```

Comment:
Seems like there's an overall lack of error handling both on the front-end + backend. You'll need to make it so that your back-end sends the correct errors so that your FE can handle them and respond as needed if something doesn't go well beyond a console.log.